test/index.js

```diff
@@ -280,14 +280,4 @@ tester.describe("parse urls", test => {
         res.user = "user";
         test.expect(res.toString()).toBe("http://user@github.com/owner/name.git");
     });
-
-    test.it("custom url", () => {
```

Comment:
Hmm, why have you removed these tests? Instead I guess we should add some tests to test what you fixed.